public/static/js/distros.js

```diff
@@ -570,6 +571,12 @@ mciModule.controller('DistrosCtrl', function($scope, $window, $http, $location,
     return false;
   }
 
+    $scope.isStatic = function() {
```

Comment:
I would just inline this in the html since it's not used elsewhere

Reply:
I looked at the HTML and it seems like we actually do a lot of static checks. I changed those usages to use isStatic rather than constantly checking `activeDistro.provider === 'static'`.